zcash_client_backend/src/wallet.rs

```diff
@@ -136,13 +136,13 @@ impl transparent_fees::InputView for WalletTransparentOutput {
 /// A subset of a [`SpendDescription`] relevant to wallets and light clients.
 ///
 /// [`SpendDescription`]: sapling::bundle::SpendDescription
-pub struct WalletSaplingSpend {
+pub struct WalletSaplingSpend<AccountId> {
     index: usize,
     nf: sapling::Nullifier,
     account: AccountId,
 }
 
-impl WalletSaplingSpend {
+impl<AccountId: Copy> WalletSaplingSpend<AccountId> {
```

Comment:
Instead of requiring the copy constraint, just return a reference from the `account` method below.